eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -0,0 +1,40 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $serviceName,
+  [Parameter(Mandatory=$True)]
+  [string] $OutDirectory
+)
+
+. (Join-Path $PSScriptRoot common.ps1)
+$allPackageProperties = @{}
+if (Test-Path "Function:Get-AllPkgProperties")
+{
+    $allPackageProperties = Get-AllPkgProperties $serviceName
+}
+else
+{
+    Write-Host "The function Get-AllPkgProperties was not found."
+}
+
+if ($allPackageProperties)
+{
+    New-Item -ItemType Directory -Force -Path $OutDirectory
+    foreach($pkg in $allPackageProperties)
+    {
+        Write-Host "Package Name: $($pkg.Name)"
+        Write-Host "Package Version: $($pkg.Version)"
+        Write-Host "Package SDK Type: $($pkg.SdkType)"
+        $outputPath = Join-Path -Path $OutDirectory ($pkg.Name + ".json")
+        $outputObject = @{name = $pkg.Name; version = $pkg.Version; SdkType = $pkg.SdkType } | ConvertTo-Json
```

Comment:
Why convert the object? I'd suggest just serializing the entire package object with all the properties as that might be useful for other scenarios.